new-cep.md

```diff
@@ -2,49 +2,37 @@
 <tr><td> Title </td><td> Conda Version Support </td>
 <tr><td> Status </td><td> Draft </td></tr>
 <tr><td> Author(s) </td><td> Travis Hathaway &lt;thathaway@anaconda.com&gt;</td></tr>
-<tr><td> Created </td><td> May 19, 2022</td></tr>
-<tr><td> Updated </td><td> May 19, 2022</td></tr>
+<tr><td> Created </td><td> March 23, 2023</td></tr>
```

Comment:
Not sure if you meant to change the creation date along with the update date.

```suggestion
<tr><td> Created </td><td> May 19, 2022</td></tr>
```

Reply:
Thanks!